<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Assignment</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumDate">
     <date>
      <year>2005</year>
      <month>3</month>
      <day>9</day>
     </date>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View Assignments</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Week View</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>763</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>763</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>763</width>
      <height>16777215</height>
     </size>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>109</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <column>
     <property name="text">
      <string>Sunday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Monday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thursday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Friday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Saturday</string>
     </property>
    </column>
   </widget>
   <zorder>calendarWidget</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>checkBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
